src/Nethermind/Nethermind.JsonRpc.Test/Modules/TraceModuleTests.cs

```diff
@@ -162,5 +166,16 @@ public void Tx_positions_are_fine()
         }
+
+        [Test]
+        public void trace_timeout_is_separate_for_rpc_calls()     
+        {
+            var searchParameter = new BlockParameter(number: 0); 
+            Assert.DoesNotThrow(() => _traceModule.trace_block(searchParameter)); 
+
+            Thread.Sleep(jsonRpcConfig.TracerTimeout + 1000); //additional second just to show that in this time span timeout should occur if given one for whole class 
```

Comment:
Default timeout is very long so it would be good to change it to something much shorter for the test.